### PR TITLE
[New Rule] Persistence via Scheduled Job Creation

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/1037

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/windows/persistence_local_scheduled_job_creation.toml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+risk_score = 47
+rule_id = "1327384f-00f3-44d5-9a8c-2373ba071e92"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Persistence"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+file where event.type != "deletion" and
+ file.path : ("C:\\Windows\\Tasks\\*.job", "D:\\Windows\\Tasks\\*.job", "E:\\Windows\\Tasks\\*.job")
```
Comment (brokensound77) [inline rules/windows/persistence_local_scheduled_job_creation.toml:24]:
```suggestion
 file.path : "?:\\Windows\\Tasks\\*" and file.extension : "job"
```

Comment (brokensound77) [reply rules/windows/persistence_local_scheduled_job_creation.toml:24]: Can we also exclude any expected `process.name`s that would be doing this

Comment (Samirbous) [reply rules/windows/persistence_local_scheduled_job_creation.toml]: on newer windows version its low volume (mostly legacy third party SW) and interesting to check by SOC analyst. I'm preparing another rule for the same that look for abnormal processes doing this kind of persistence (which should have high severity/low FP and may go for endpoint-rules).

File: rules/windows/persistence_local_scheduled_job_creation.toml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+[metadata]
+creation_date = "2021/03/15"
+maturity = "production"
+updated_date = "2021/03/15"
+
+[rule]
+author = ["Elastic"]
+description = "A scheduled job can be used by an adversary to establish persistence."
```
Comment (bm11100) [inline rules/windows/persistence_local_scheduled_job_creation.toml:8]:
```suggestion
description = "A job can be used to schedule programs or scripts to be executed at a specified date and time. Adversaries may abuse task scheduling functionality to facilitate initial or recurring execution of malicious code."
```

**Review by bm11100 — APPROVED** [review]

**Review by brokensound77 — APPROVED** [review]
LGTM 👍

**Pull request merged**